_translator-files/po/de/Installation-for-Windows.po

```diff
@@ -98,12 +99,12 @@ msgstr "**Richte deine Soundkarte ein**. Wenn du fertig bist, musst du deine Aud
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:24
 msgid "_Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._"
-msgstr ""
+msgstr "_Bitte beachte, dass du diese Binärdatei nicht weitergeben darfst, ohne eine [Lizenzvereinbarung von Steinberg](https://www.steinberg.net/developers/) zu erwerben_"
```

Comment:
Not erwerben but beantragen or something similar (erwerben involves money, but it's free)

Reply:
Corrected